webpack.mix.js

```diff
@@ -0,0 +1,15 @@
+const mix = require('laravel-mix');
```

Comment:
I think it's perfect, I just left it because it's what comes by default with Laravel and since it wasn't being used "it didn't bother".

I remove it and I can't wait to see the scripts and the css in folders separated by modules and then everything optimized and minified :)